packages/devextreme/testing/tests/DevExpress.ui.widgets/multiView.tests.js

```diff
@@ -124,6 +124,41 @@ QUnit.module('rendering', () => {
         }
     });
 
+    QUnit.test('multiview items.visible option should always be default to true', function(assert) {
```

Comment:
[naming]
The test name described not what u're checking here. U say "option should be true" but checks that some class is presented.
```suggestion
    QUnit.test('item should be visible if no item.visible property is specified', function(assert) {
```